src/OrchardCore.Modules/OrchardCore.Placements/Views/Admin/Edit.cshtml

```diff
@@ -0,0 +1,76 @@
+@model EditShapePlacementViewModel
+@{
+    var returnUrl = ViewData["returnUrl"]?.ToString();
+}
+
+<style asp-name="codemirror"></style>
+<script asp-name="codemirror" depends-on="admin" at="Foot"></script>
+<script asp-name="codemirror-addon-display-autorefresh" at="Foot"></script>
+<script asp-name="codemirror-addon-edit-closetag" at="Foot"></script>
+<script asp-name="codemirror-addon-selection-active-line" at="Foot"></script>
+<script asp-name="codemirror-mode-css" at="Foot"></script>
```

Comment:
and here you can probably get rid of the closetag, css and html / xml formatters. just need json right ? which I think is covered by the javascript formatter